lib/lz4frame.h

```diff
@@ -160,6 +160,11 @@ typedef enum {
     LZ4F_OBSOLETE_ENUM(skippableFrame)
 } LZ4F_frameType_t;
 
+typedef enum {
```

Comment:
Is this `enum` used / manipulated by the user ?
If not, it doesn't need to be part of the public API, 
and can remain private inside `lz4frame.c`.

Reply:
Moved it into `lz4frame.c` as it's not supposed to by used by the user.